src/style.css

```diff
@@ -1,3 +1,74 @@
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  font-size: 12px;
+  font-weight: 500;
+  font-family: Roboto, sans-serif;
+
+  --main-color: rgba(0, 172, 220, 1);
```

Comment:
Create a variable with root selector

Reply:
Still not fixed from the previous review, you need to create variable with root selector
https://www.w3schools.com/css/css3_variables.asp
<img width="910" alt="image" src="https://github.com/mate-academy/layout_moyo-header/assets/64750958/03a73e44-972a-469b-901d-ab5070db787c">